Documentation/project-docs/developer-guide.md

```diff
@@ -13,11 +13,12 @@ In order to build .NET Command Line Interface, you need the following installed
 
 1. git (available from http://www.git-scm.com/) on the PATH.
 
-### For OS X
+### For macOS (OS X)
 
 1. Xcode
 2. git (available from http://www.git-scm.com/) on the PATH.
 3. Install OpenSSL (a .NET Core requirement)
+  - Install Homebrew (available from https://brew.sh/).
```

Comment:
I think we can remove this entire step because .NET Core 2.0+ uses Apple Security Framework: https://github.com/dotnet/announcements/issues/21